tests/conftest.py

```diff
@@ -111,6 +191,54 @@ def kb_one():
     })[0]
 
 
+@pytest.fixture
+def kb_four():
+    """Return a KojiBuild matching mock_getBuild_four."""
+    return KojiBuild.get_or_create({
+        'completion_time': datetime(2018, 6, 15, 20, 26, 38, tzinfo=pytz.utc),
+        'creation_time': datetime(2018, 8, 20, 17, 32, 24, tzinfo=pytz.utc),
+        'id_': '754575',
+        'name': 'podman',
+        'owner_name': 'emusk',
+        'release': '1.git9d09a4d.el8',
+        'start_time': datetime(2018, 8, 20, 17, 32, 24, tzinfo=pytz.utc),
+        'state': 0,
+        'version': '0.8.3'
+    })[0]
+
+
+@pytest.fixture
+def kb_canceled():
+    """Return a KojiBuild matching mock_getBuild_four."""
+    return KojiBuild.get_or_create({
+        'completion_time': datetime(2018, 8, 18, 15, 4, 47, tzinfo=pytz.utc),
+        'creation_time': datetime(2018, 8, 15, 13, 47, 43, tzinfo=pytz.utc),
+        'id_': '751869',
+        'name': 'devtoolset-8-gcc',
+        'owner_name': 'emusk',
+        'release': '2.2.el7',
+        'start_time': datetime(2018, 8, 15, 13, 47, 43, tzinfo=pytz.utc),
+        'state': 4,
+        'version': '8.2.1'
+    })[0]
+
+
+@pytest.fixture
+def kb_deleted():
+    """Return a KojiBuild matching mock_getBuild_four."""
+    return KojiBuild.get_or_create({
+        'completion_time': datetime(2017, 12, 7, 22, 32, 47, tzinfo=pytz.utc),
+        'creation_time': datetime(2017, 12, 7, 22, 21, 54, tzinfo=pytz.utc),
+        'id_': '623989',
+        'name': 'gnome-terminal',
+        'owner_name': 'emusk',
+        'release': '1.el8',
+        'start_time': datetime(2017, 12, 7, 22, 21, 54, tzinfo=pytz.utc),
+        'state': 1,
```

Comment:
This is confusing since the fixture is called `kb_deleted` but this build is in the completed state.